### ISSUE TITLE
Support Selenium 3.5.x

### ISSUE THREAD

**Issue by aaronrenner** [keathley/wallaby#305]
It appears that selenium 3.5.x has changed their API responses and now we're getting test failures like this:

```
11) test visible?/1 handles elements that are not on the page (Wallaby.Integration.Browser.VisibleTest)
     integration_test/cases/browser/visible_test.exs:19
     ** (FunctionClauseError) no function clause matching in Wallaby.Experimental.Selenium.WebdriverClient.cast_as_element/2

     The following arguments were given to Wallaby.Experimental.Selenium.WebdriverClient.cast_as_element/2:

         # 1
         %Wallaby.Session{driver: Wallaby.Experimental.Selenium, id: "14b714ac-8572-2c4d-b651-1100bf2c23ac", screenshots: [], server: :none, session_url: "http://localhost:4444/wd/hub/session/14b714ac-8572-2c4d-b651-1100bf2c23ac", url: "http://localhost:4444/wd/hub/session/14b714ac-8572-2c4d-b651-1100bf2c23ac"}

         # 2
         %{"element-6066-11e4-a52e-4f735466cecf" => "1bb466ca-cca9-9747-a068-0296c171431c"}

     Attempted function clauses (showing 1 out of 1):

         defp cast_as_element(parent, %{"ELEMENT" => id})

     code: element = find(page, Query.css("#off-the-page", visible: false))
     stacktrace:
       (wallaby) lib/wallaby/experimental/selenium/webdriver_client.ex:358: Wallaby.Experimental.Selenium.WebdriverClient.cast_as_element/2
       (elixir) lib/enum.ex:1255: Enum."-map/2-lists^map/1-0-"/2
       (wallaby) lib/wallaby/experimental/selenium/webdriver_client.ex:35: Wallaby.Experimental.Selenium.WebdriverClient.find_elements/2
       (wallaby) lib/wallaby/browser.ex:913: anonymous fn/3 in Wallaby.Browser.execute_query/2
       (wallaby) lib/wallaby/browser.ex:142: Wallaby.Browser.retry/2
       (wallaby) lib/wallaby/browser.ex:454: Wallaby.Browser.find/2
  ```

I didn't see anything in the [selenium changelog](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/CHANGES.md), but it seems like they've started putting UUIDs in the element responses. I've confirmed that everything still works fine on Selenium 3.4.0.

**Comment by keathley** [comment]
Is there a clean way of supporting selenium 3.4 and 3.5? Or does this mean we would start to drop selenium 3.4?

**Comment by keathley** [comment]
@aaronrenner Is this still something that we need to address or can we close this issue for now?

**Comment by mhanberg** [comment]
This was addressed in #392
